### PR TITLE
Host images with FlowForge

### PR DESCRIPTION
**Pull request by robmarcer**
## Description

An article explaining how to use images in dashboard hosted on FlowForge.

### CODE REVIEW

**Comment by robmarcer** [comment]
@iskerrett

**Comment by robmarcer** [comment]
This still needs the image header adding - https://github.com/flowforge/content/issues/123

File: src/blog/2023/07/images/the-dashboard.gif
Comment (ZJvandeWeg) [inline src/blog/2023/07/images/the-dashboard.gif:1]: Love the gif!

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Add images to dashboards when working in containerised Node-RED
```
Comment (iskerrett) [inline src/blog/2023/07/images-in-node-red-dashboards.md:2]:
```suggestion
title: How to add images to Node-RED dashboards when using FlowForge?
```

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+1. Serve up those images as needed in the dashboard
+
+It is possible for us to skip step 2, but I wanted to have the images stored locally, in the Node-RED instance. storing the images locally will improve the loading times of the dashboard. This is especially beneficial when your dashboard is dynamically displaying relevant images, e.g. to show an image of a specific machine which needs to be attended to.
+
+The key benefit of pulling the images from URLs this way is, no matter where you are running Node-RED, the correct images will be shown in your dashboard.
+
+## File and file-in nodes
+
+I've included the flows as json below so you can try them out yourself. Please note, I'm using FlowForge's own [file and file-in nodes](https://flowforge.com/docs/user/filenodes/) in these examples. If you want to use these flows on hosting other than FlowForge, you will need to replace the nodes with the standard Node-RED file and file-in nodes.
```
Comment (ZJvandeWeg) [inline src/blog/2023/07/images-in-node-red-dashboards.md:52]:
```suggestion
I've included the flows as json below so you can try them out yourself. Please note, I'm using FlowForge's own [file and file-in nodes](/docs/user/filenodes/) in these examples. If you want to use these flows on hosting other than FlowForge, you will need to replace the nodes with the standard Node-RED file and file-in nodes.
```

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## More things to try
+
+In this example, the images are static but it's simple to load images depending on the state of the flow. As mentioned in this article's introduction, you could display context aware images guiding the user of the dashboard to a specific location on a map, to complete a maintenance task. If you're interested in seeing examples of dynamic image loading please comment below.
+
+## Conclusion
+
+Images can add a lot of value to dashboards, but depending on how you host a given Node-RED instance it can be challenging to ensure the correct images are shown, especially when working with your flows in a DevOps pipeline. The techniques I've discussed above allow you to use images in dashboards, even in containerised environments. 
+
+I'd love to hear your comments and suggestions on this article. please tell us what you think about this article, and how you might use these techniques in the comments section below.
```
Comment (ZJvandeWeg) [inline src/blog/2023/07/images-in-node-red-dashboards.md:93]: Thanks :)

**Comment by ZJvandeWeg** [comment]
Pending the tile from @Yndira-FlowForge, but otherwise; feel free to merge.

**Pull request merged**